apps/daimo-mobile/src/view/screen/send/SendNavScreen.tsx

```diff
@@ -20,33 +20,34 @@ type Props = NativeStackScreenProps<ParamListSend, "SendNav">;
 export function SendNavScreen({ route }: Props) {
   const { autoFocus } = route.params || {};
 
+  // Search prefix
+  // Clear prefix on back button
```

Comment:
snuck in this bugfix:
- click send
- enter a letter, see search results
- click back
- click send again

before: your old search prefix is still there. there is no way to clear out of search without manually backspacing away the query.

new: query is cleared.